docs/USAGE-GUIDE.md

```diff
@@ -762,10 +762,27 @@ or if called on a connection in a bad state.
 
 Once `s2n_shutdown()` is complete:
 * The s2n_connection handle cannot be used for reading or writing.
-* The underlying transport can be closed, most likely via `close()`.
+* The underlying transport can be closed, most likely via `shutdown()` or `close()`.
 * The s2n_connection handle can be freed via `s2n_connection_free()` or reused
 via `s2n_connection_wipe()`
 
+#### Closing the connection for writes
+
+TLS1.3 supports closing the write side of a TLS connection while leaving the read
```

Comment:
Why would someone want to do this?

Reply:
Tried to add more context to address this